core/src/test/java/net/sourceforge/jnlp/config/DeploymentConfigurationTest.java

```diff
@@ -81,6 +85,30 @@ public void testInstallProperties() throws ConfigurationException {
         assertTrue(!target.isEmpty());
     }
 
+    @Test
+    public void loadEmptyDeploymentPropertiesFile() throws ConfigurationException, IOException {
+        File f = File.createTempFile("emptyDeployment", "properties");
```

Comment:
can be made `final` (also in the tests below)

Reply:
done